python/pyspark/sql/connect/functions/builtin.py

```diff
@@ -3804,12 +3804,14 @@ def sha2(col: "ColumnOrName", numBits: int) -> Column:
 sha2.__doc__ = pysparkfuncs.sha2.__doc__
 
 
-def hll_sketch_agg(col: "ColumnOrName", lgConfigK: Optional[Union[int, Column]] = None) -> Column:
+def hll_sketch_agg(
+    col: "ColumnOrName",
+    lgConfigK: Optional[Union[int, Column]] = None,
+) -> Column:
     if lgConfigK is None:
         return _invoke_function_over_columns("hll_sketch_agg", col)
     else:
-        _lgConfigK = lit(lgConfigK) if isinstance(lgConfigK, int) else lgConfigK
```

Comment:
This is a logical code change, isn't it? If the previous code is a bug, please file a JIRA and handle it separately, @zhengruifeng .

Reply:
oh, it is not a logical change.
The `lit` function actually accepts a `Column` input and return the column itself, so for a `Union[int, Column]` input, we don't not need this `if else` (there should be other similar cases, I just update here BTW)

Reply:
I add this change to the PR desriptions